assignments/closure.js

```diff
@@ -1,11 +1,31 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+function GetDressed() {
+  let character = "Knight";
+
+  function wearHelmet() {
+    let helmet = "Lambda Cap"
+    console.log(character + " dons their " + helmet);
+    function grabShield() {
+      console.log(character + " won't wear their " + helmet + " without grabbing their shield as well!");
+    }
+    grabShield();
+  }
+  wearHelmet();
+}
+GetDressed();
+
 
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
   // Return a function that when invoked increments and returns a counter variable.
-};
+  let i = 0;
+  return function() { return i++;}
```

Comment:
Your closure is logging the count starting at 0. If you want it to start at 1, you could use `{return ++i}`.